cli/cmd/integration.go

```diff
@@ -279,6 +280,8 @@ func promptCreateIntegration() error {
 		return createAwsConfigIntegration()
 	case "AWS CloudTrail":
 		return createAwsCloudTrailIntegration()
+	case "AWS Gov Cloud Config":
```

Comment:
It is AWS GovCloud: https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-differences.html
```suggestion
	case "AWS GovCloud Config":
```